src/parser/jobs/sam/modules/Sen.tsx

```diff
@@ -110,4 +319,78 @@ export default class Sen extends Module {
 			why: <Trans id = "sam.sen.suggestion.why">You wasted {this.wasted} sen.</Trans>,
 		}))
 	}
+
+	output() {
+		return <Fragment>
+			<Message>
+				<Trans id="sam.sen.rotation-table.message"> This table serves as a way a way to better see the events that lend up to a Sen window that has been deemed "Non-Standard" as explained below. Dying, overwriting a sen, or using hagakure will cause a window to be flagged as Non-Standard.
+				</Trans>
+			</Message>
+				<Message warning icon>
+					<Icon name="warning sign"/>
+					<Message.Content>
+						<Trans id = "sam.sen.rotation-table.disclaimer">This module labels a "Standard Sen Window" to be a window with no Sen overwrites that ends on a Iaijutsu. Please consult The Balance Discord's guides for more details on looping Samurai gameplay.
```

Comment:
Nit: "ends on **an** Iaijutsu

Also, is there a specific Balance guide we can link to in the recommendation?

Reply:
Fixed.